storybookjs/aem
# Clientlib loading part 1

## Issue: #14 

## What I did
Instead of merging this whole feature in at once, Ive decided to do it in chunks. The code added in this PR detects the clientlib js/css/less files and auto-includes them (only those for a specific component) in the iframe during the component loading process. Im currently working on the followup parts which include detecting categories and dependencies. After this PR is merged, we will no longer need to require the clientlib files for a specific component individually as reflected in the documentation update.

## How to test

- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.
